src/main/java/com/checkmarx/sdk/utils/scanner/client/ScaClientHelper.java

```diff
@@ -623,15 +623,25 @@ public String createRiskManagementProject(String name) throws IOException {
     }
 
     private void determineProjectTeam(CreateProjectRequest request) {
-        String team = scaProperties.getTeamForNewProjects();
+        String team = config.getScaConfig().getTeam();
```

Comment:
1. For better readability, you can use
```suggestion
        String team = StringUtils.firstNonEmpty(config.getScaConfig().getTeam(), scaProperties.getTeam());
```
And then call setTeam(request, team) or request.setAssignedTeams(null) accordingly.

2. Also I suggest adding some logging.

Reply:
Nice. I update the code.